projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -288,6 +288,16 @@ function wpcom_launchpad_get_task_definitions() {
 			},
 			'is_complete_callback' => 'wpcom_is_task_option_completed',
 		),
+		'write_posts'                     => array(
```

Comment:
I think we should include the number of posts in the slug to make the number of posts as explicit as possible, as we may have situations where we want to prompt users to create other numbers of posts.
```suggestion
		'write_3_posts'                     => array(
```